pkg/etc/wavefront/wavefront-proxy/wavefront.conf.default

```diff
@@ -3,83 +3,60 @@
-#  We strongly recommend setting this to a name that is unique among your entire infrastructure,
-#   possibly including the datacenter information, etc. This hostname does not need to correspond to
-#   any actual hostname or DNS entry; it's merely a string that we pass with the internal stats.
-#
+# We strongly recommend setting this to a name that is unique among your entire infrastructure, to make this
+# proxy easy to identify. This hostname does not need to correspond to any actual hostname or DNS entry; it's merely
+# string that we pass with the internal stats and ~proxy.* metrics.
 #hostname=my.proxy.host.com
 
-# The Token is any valid API Token for your account, which can be generated from the gear icon
-#   at the top right of the Wavefront site, under 'Settings'. Paste that hexadecimal token
-#   after the '=' below, and the proxy will automatically generate a machine-specific UUID and
-#   self-register.
-#
-#token=XXX
-
-## Set to true when running proxy inside containers or when the proxy is frequently restarted on a new infrastructure.
-## When true, terminated proxies will be automatically removed from the UI after 24 hours of inactivity.
-#ephemeral=false
+# The Token is any valid API Token for your account that has "Proxy Management" permissions. This token can be
+# by clicking on the gear icon at the top right of the Wavefront site -> click on your email -> click on "API access".
+#token=
 
-# Comma separated list of ports to listen on for Wavefront formatted data
+####################################################### INPUTS #########################################################
+# Comma separated list of ports to listen on for Wavefront formatted data (Default: 2878)
 pushListenerPorts=2878
-## Comma separated list of ports to listen on for OpenTSDB formatted data
-#opentsdbPorts=4242
-## Comma separated list of ports to listen on for HTTP JSON formatted data
-#jsonListenerPorts=3878
-## Comma separated list of ports to listen on for HTTP collectd write_http data
-#writeHttpJsonListenerPorts=4878
-## Comma separated list of ports to listen on for Graphite pickle formatted data (from carbon-relay)
-#picklePorts=5878
-
-## Which ports should listen for collectd/graphite-formatted data?
-## If you uncomment graphitePorts, make sure to uncomment and set 'graphiteFormat' and 'graphiteDelimiters' as well.
+## Maximum line length for received points in plaintext format on Wavefront/OpenTSDB/Graphite ports. Default: 32KB
+#pushListenerMaxReceivedLength=32768
+## Maximum request size (in bytes) for incoming HTTP requests on Wavefront/OpenTSDB/Graphite ports. Default: 16MB
+#pushListenerHttpBufferSize=16777216
+
+## Graphite input settings.
+## If you enable either graphitePorts or picklePorts, make sure to uncomment and set 'graphiteFormat' as well.
+## Comma separated list of ports to listen on for collectd/Graphite formatted data (Default: none)
```

Comment:
...and set 'graphiteFormat' ... 
=> and set `graphiteFormat` 
(Use backticks, not single quotes)